src/lib/components/Paper.react.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import React, { Component } from 'react';
+
+import MuiPaper from '@material-ui/core/Paper';
+import lightBaseTheme from 'material-ui/styles/baseThemes/lightBaseTheme';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+
+type Props = {
+  /** ID for Paper */
+  id?: string,
+  /** Can be used to render elements inside the Paper*/
+  children?: Node,
+  /** The CSS class name of the root element */
+  className?: string,
+  /** The classes to be applied to this component. This keys in this object must be valid CSS rule
+   * names, and the values must be strings for the classnames to be assigned to each rule name */
+  classes?: Object,
```

Comment:
Please add full set of options with type hinting.

Reply:
Paper had a strange set of props, see if the documentation I left makes sense to you